### DISCUSSION TITLE
how to make loadbalancer connect to public network when loadbalancer created

### DISCUSSION THREAD

**Discussion by akalitenya** [vultr/vultr-cloud-controller-manager#309] [General]
loadbalancer picks different network when loadbalancer created

**Comment by optik-aper** [comment]
@akalitenya Would the `vpc` annotation get you what you're trying to do?

https://github.com/vultr/vultr-cloud-controller-manager/blob/master/docs/load-balancers.md

**Comment by akalitenya** [comment]
I tried vpc annotation before creating issue, but as I remember newly created loadbalancer still appeared to be connected to some vpc (not public network). So I changed it through UI. But as I understand, UI changes may not persist If I upgrade helm chart related to loadbalancer for example. It would be nice if I able to setup everything consistently through helm charts, without need for manual changes in UI.
